lib/widget/listwidget.h

```diff
@@ -90,14 +99,14 @@ class ListWidget : public WIDGET
 public:
 	enum Order {RightThenDown, DownThenRight};
 
-	ListWidget(WIDGET *parent);
+	ListWidget();
 
-	void widgetLost(WIDGET *widget) override;
+	void widgetLost(std::shared_ptr<WIDGET> &widget) override;
```

Comment:
Probably should either be `const std::shared_ptr<WIDGET> &widget` or `std::shared_ptr<WIDGET> widget`

Reply:
Changed to const.